binderhub/build.py

```diff
@@ -255,7 +274,7 @@ def submit(self):
                         volume_mounts=volume_mounts,
                         resources=client.V1ResourceRequirements(
                             limits={'memory': self.memory_limit},
-                            requests={'memory': self.memory_limit}
+                            requests={'memory': self.memory_request},
```

Comment:
The travis error is about `self.memory_request` not existing: `AttributeError: 'Build' object has no attribute 'memory_request'` I think we need to assign it in the constructor as it is new and `Build` doesn't use traitlets

Reply:
Fixed, thanks! I write so much in traitlets-land now that I've forgotten how to write normal Python constructors...

Reply:
Should we make Build a Configurable? Lots of BinderHub's configuration is ultimately only for Build, and we need to do quite a bit of passing around through settings, constructors, etc. because where it's exposed as config (BinderHub) is not where it's used (Build).

Reply:
Moving to a configurable sounds like a good idea. Only caveat is that I don't know why it isn't already a configurable and maybe it was a on purpose decision?